diff --git a/dist/index.mjs b/dist/index.mjs
index 04537d45037445e8dc497d90ad631e48af69a18d..4d5ea3ef4c39e1c3cd1f5a562fd16db78cbd85fe 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -3538,7 +3538,7 @@ const schemaModule = defineNuxtModule({
       return;
     }
     const resolver = createResolver(import.meta.url);
-    const _resolveSchema = jiti(dirname(import.meta.url), {
+    const _resolveSchema = jiti(dirname(fileURLToPath(import.meta.url)), {
       esmResolve: true,
       interopDefault: true,
       cache: false,
